prow/gcsupload/options.go

```diff
@@ -63,6 +63,8 @@ type Options struct {
 	gcsPath gcs.Path
 }
 
+const defaultMaxConcurrency int64 = 50
```

Comment:
https://github.com/GoogleCloudPlatform/testgrid/blob/e150868219d53c2e3cbc56310e5bb9dc8a3a4336/cmd/updater/main.go#L62 is what I've been using (cpus * 4)

Reply:
Ooh, I like that. Do you have any clue what it returns in a container in k8s under a CPU-limited cgroup?

Reply:
No clue, but seems to have always provided me sufficiently fast results